packages/api-client/src/models/manifestPreview.ts

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable max-classes-per-file */
```

Comment:
Letting these pass because we can try to isolate these later. Maybe we should add these as issues.

Reply:
Having this file divided up into 6 separate files to having each one with a class that inherits from a base and adds 1 thing to it does seem to be a bit excessive, but we can address this later.